### ISSUE TITLE
Removal of Pixels.PhysicalSizeXUnit and .PhysicalSizeYUnit

### ISSUE THREAD

**Issue by keller-mark** [tlambert03/ome-types#144]
Hi,
I have generated an OME-TIFF using the following command:

```sh
bfconvert -tilex 512 -tiley 512 -pyramid-resolutions 6 -pyramid-scale 2 -compression LZW my_image.tiff my_image.ome.tif
```

The output of 
```sh
~/software/bftools/tiffcomment my_image.ome.tif > tiffcomment_out.ome.txt
```
is [tiffcomment_out.ome.txt](https://github.com/tlambert03/ome-types/files/9065307/tiffcomment_out.ome.txt)

However, the output of

```py
from ome_types import from_tiff, to_xml
with open("ome_types_out.ome.txt", "wt+") as f:
  f.write(to_xml(from_tiff("my_image.ome.tif")))
```
is [ome_types_out.ome.txt](https://github.com/tlambert03/ome-types/files/9065318/ome_types_out.ome.txt)

The `ome_types` output erases the properties Pixels.PhysicalSizeXUnit and Pixels.PhysicalSizeYUnit. I am using a downstream tool that is relying on these properties to be in the OME-XML.

Is this a bug or is this the expected behavior?

**Comment by tlambert03** [comment]
It's expected in a sense, since the values in your data are just the default values for `Pixels. PhysicalSizeYUnit ` in the [OME model](https://www.openmicroscopy.org/Schemas/Documentation/Generated/OME-2016-06/ome.html), and when serialize OME objects we exclude default values.  Given the huge size of the model, `bfconvert` is _also_ clearly excluding a lot of the default values, but i'm, not sure what logic they are using to preferentially include specific default values like `Pixels.PhysicalSizeXUnit = 'µm'` ... We can expose some additional parameters in `to_xml` that would let you tune what to include

**Comment by tlambert03** [comment]
(re Channel.Color = "-1" ... that is now solved in #164)

**Comment by tlambert03** [comment]
The next version (v0.4) will be better about this in general.  `to_xml()` gains two arguments:
- exclude_defaults : bool, optional
    Whether to exclude attributes that are set to their default value,
    by default False.
- exclude_unset : bool, optional
    Whether to exclude attributes that are not explicitly set,
    by default True.

The default of `exclude_unset=True` will work great for your case, if `Pixels.PhysicalSizeXUnit` was present in the incoming xml, it will be present in the outgoing xml.  To _force_ everything to be included, you could use `exclude_unset=False` and `exclude_defaults=False`
